references.bib

```diff
@@ -1,3 +1,15 @@
+@book{df80bc12-en,
+   author = "OECD",
+   title = "OECD Skills Outlook 2019",
+   year = "2019",
+   pages = 285,
+   url = "https://www.oecd-ilibrary.org/content/publication/df80bc12-en",
+   doi = "https://doi.org/https://doi.org/10.1787/df80bc12-en" 
```

Comment:
This DOI is misformed - Also, I think we should just provide a DOI and not the URL as well unless there is no DOI, otherwise we end up with two links to the same thing

Reply:
ok. I'll remove the URL>